### ISSUE TITLE
Powerline not reconnecting after conn loss

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#731]
Part of https://github.com/kinu-garage/30y-130s_life/issues/1545
Issue
- LAN network via Ethernet continues to be disconnected in my office, after Powerline disconnection (, of which trigger is unknown).
- It is not obvious the Powerline connection was lost. Had to look at the status light on the Powerline client device (this is inconvenient).

Req
- No human operation needed to maintain network connection after an inevitable connection loss.
- I get push notification when the network is lost for longer than certain period.

**Comment by 130s** [comment]
Not automatically reconnecting may or may not be an inherent problem my current Powerline product. I won't try to resolve this issue for now, as I don't know how long this will take to figure out.

While I like to keep Powerline as an option, I might push it aside to the usecases that incurs less criticality. Looking into other options.

https://zappedia.com/wifi-to-ethernet-bridge/

"Wifi bridge" sounds like what I've been thinking of.

Deco device https://github.com/kinu-garage/30y-130s_life/issues/1057#issuecomment-1011832150 might be able to bridge the wifi. It has ethernet port. Needs to verify it works in this way.

**Comment by 130s** [comment]
I added another Deco device, linked it to the existing Wifi mesh network that other Deco devices created, then connected an Ethernet cable from the new Deco to the existing dumb hub (which I suspected can be one of the bottlenecks). Result is a blast! [405 Mbps DL 41 Mbps UP](https://www.speedtest.net/result/13954756976). So...this tells me the single bottleneck was the Powerline.

I'll use Powerline product in other parts of the house as mentioned.
